sdk/test/_metrics/exact_aggregator_test.cc

```diff
@@ -83,7 +83,7 @@ TEST(ExactAggregatorOrdered, Types)
   // This test verifies that we do not encounter any errors when
   // using various numeric types.
   ExactAggregator<int> agg_int(metrics_api::InstrumentKind::Counter);
-  ExactAggregator<long> agg_long(metrics_api::InstrumentKind::Counter);
+  ExactAggregator<int64_t> agg_long(metrics_api::InstrumentKind::Counter);
```

Comment:
These are for old metrics implementation :)

Reply:
Thanks, reverted old metrics 😄